pkg/core/config/main.go

```diff
@@ -175,7 +175,7 @@ func New(option Option) (config Config, err error) {
 	}
 
 	/** Check for deprecated directives **/
-	// pullequests deprecated over actions
+	// pull requests deprecated over actions
```

Comment:
```suggestion
	// pullrequests deprecated over actions
```

Imho in this case, it's the Updatecli "pullrequests" concept that is depecated in favor of "actions"